app/models/imo_authorization.rb

```diff
@@ -0,0 +1,12 @@
+class ImoAuthorization < ApplicationRecord
+  belongs_to :patient
+  validates :last_invited_at, presence: true
+  validates :status, presence: true
+
+  enum status: {
+    invited: "invited",
```

Comment:
`invited` just means it has been sent, right?

Reply:
Yes.